R/connect.R

```diff
@@ -163,15 +163,17 @@ connectClient <- function(service, authInfo) {
       while (TRUE) {
         path <- paste0(file.path("/tasks", taskId), "?first_status=", start)
         response <- handleResponse(GET(service, authInfo, path))
+        messages <- unlist(response$status)
 
-        labeledMessage <- function(msg) {
-          message(paste("[Connect]", msg))
-        }
+        # Strip timestamps, if found
+        timestamp_re <- "\\d{4}/\\d{2}/\\d{2} \\d{2}:\\d{2}:\\d{2}\\.\\d{3,} "
```

Comment:
The `[Connect]` preamble was trying to make it more clear what information was produced on the client and what information was sent back from the server. Do you feel like this separation is still clear?

Reply:
I think so, because the rules help delineate the output from connect:

<img width="667" alt="Screenshot 2023-02-28 at 09 01 21" src="https://user-images.githubusercontent.com/4196/221892648-09066351-1ffe-4e61-9e06-734b4446b42d.png">

Maybe the rule could have a more informative title? I can't think what it should say though.